core/ui/mpc/session/StartMpcSessionStep.tsx

```
@@ -14,6 +14,8 @@ import { useEffect } from 'react'
 import { useTranslation } from 'react-i18next'
 
 import { MpcSession } from './MpcSession'
+import { hasServer, isServer } from '@core/mpc/devices/localPartyId'
+import { useCurrentKeygenType } from '../keygen/state/currentKeygenType'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)